content/en/docs/concepts/storage/volumes.md

```diff
@@ -49,8 +49,8 @@ the {{< glossary_tooltip text="container image" term_id="image" >}}, plus volume
 (if defined) mounted inside the container.
 The process sees a root filesystem that initially matches the contents of the container
 image.
-Any writes to within that filesystem hierarchy, if allowed, affect what that process views
-when it performs a subsequent filesystem access.
+Any writes within that filesystem hierarchy, if allowed, affects what that process views
```

Comment:
Changes look good to me.

Deploy preview: https://deploy-preview-39503--kubernetes-io-main-staging.netlify.app/docs/concepts/storage/volumes/